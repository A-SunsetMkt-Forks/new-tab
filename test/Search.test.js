'use strict';

const Search = require('../src/components/Search.html');

describe('Search component', () => {
  it('renders correctly', () => {
    const target = document.createElement('div');
    new Search({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
