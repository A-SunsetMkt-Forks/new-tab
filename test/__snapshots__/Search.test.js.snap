// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders correctly 1`] = `
"<div class=\\"c\\"><input id=\\"s\\" placeholder=\\"Search in browser...\\" autocomplete=\\"off\\">

  
  <p class=\\"g\\">No matching Open Tabs</p>

  
  

  
  

  
  </div>"
`;
